xclim/testing/helpers.py

```diff
@@ -195,3 +200,36 @@ def add_example_file_paths(cache_dir: Path) -> dict[str]:
             ns[f"{variable}_dataset"] = ds.get(variable)
 
     return ns
+
+
+def test_timeseries(
+    values, variable, start="7/1/2000", units=None, freq="D", as_dataset=False
+):
+    coords = pd.date_range(start, periods=len(values), freq=freq)
+    data_on_var = safe_load(open_text("xclim.data", "variables.yml"))["variables"]
+    if variable in data_on_var:
+        attrs = {
+            a: data_on_var[variable].get(a, "unknown")
+            for a in ["description", "standard_name", "cell_methods"]
+        }
+        attrs["units"] = data_on_var[variable]["canonical_units"]
+
+    else:
+        warnings.warn(
+            f'Variable {variable} not recognised. Attrs will filled with "unknown".'
+        )
+        # TODO: is it better to put this or just no attrs ?
+        attrs = {
+            a: "unknown"
+            for a in ["description", "standard_name", "cell_methods", "units"]
+        }
```

Comment:
It depends on whether this is used in Indicators. I would opt to remove it for now and see what breaks.